### ISSUE TITLE
Where to find the Classification part?

### ISSUE THREAD

**Issue by twins14** [experiencor/keras-yolo2#240]
hello, with Full Yolo backend the network consist of a classification part and a detection part. Every Classification Part has at least one Fully Connected Layer with softmax at the end but i cannot find such Layer in the code?

**Comment by experiencor** [comment]
@twins14 Your assumption about Classification Part is not right. Fully Connected Layer with software is replaced by the Detection Part.

**Comment by bexx1963** [comment]
That means you are not using any Dense Layer?
I would like to use the categorical_crossentropy instead your custom_loss function. For that i need as input a probability distribution. And a Dense Layer would give me that distribution

**Comment by experiencor** [comment]
No, there's no dense layer as this is YOLOv2. YOLOv1 does have 2 dense layers at the end.
